package.json

```diff
@@ -21,7 +21,7 @@
     "pancake <pancake@nopcode.org>"
   ],
   "name": "radare2-git",
-  "version": "1.6.0-git",
+  "version": "4.5.1-git",
```

Comment:
thats not what configure.acr says. right now this is 4.6.0-git. anyway its a pain to update this file version. this can be done with a jq oneliner or nodejs. and should be good to npm pub at some point.. or well decide that npm is probably not the place to distribute r2. But as long as r2 is compiled with emscripten and wasm. this node module aims to provide a way to use r2 from nodejs with pure JS, no native things involved. So imho its fine to merge this if the version is the correct one, but its totally low prio unless we do emscripten builds in CI